packages/optimizer/lib/transformers/AutoExtensionImporter.js

```diff
@@ -141,12 +141,7 @@ class AutoExtensionImporter {
       this.log_.error('Missing validation rules, cannot auto import extensions');
       return;
     }
-    if (!this.bentoComponentInfo) {
-      this.bentoComponentInfo = {};
-      for (const bentoComponent of params.bentoComponentInfo || []) {
-        this.bentoComponentInfo[bentoComponent.name] = bentoComponent;
-      }
-    }
+    this.componentVersions = params.componentVersions || {};
```

Comment:
Since optional chaining is supported in node 14￼, Would you ever want to use it rather than this type of syntax? Or do you prefer this type of format￼

Reply:
How does optional chaining help here? Do you mean nullish coalescing operator?